Lesson4HomeWork.playground/Lesson4HomeWork.playground/Contents.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+let firstTupple: (Int, String) = (0, "zero")
+let secondTupple = (2, "two")
+let thirdTupple = (3, "three")
+let fourthTupple = (4, "four")
+let fifthTupple = (5, "five")
+let sixthTupple = (6, "six")
+let seventhTupple = (7, "seven")
+let eigthsTupple = (8, "eigth")
+
+var arrayTupples = [firstTupple, secondTupple, thirdTupple, fourthTupple, fifthTupple, sixthTupple, seventhTupple, eigthsTupple]
+var powArrayTupples = arrayTupples
+
+for elements in 0..<powArrayTupples.count {
+    let powNumbersArray = [(Int(pow(Double(powArrayTupples[elements].0), 2)), powArrayTupples[elements].1)] as [Any]
+    print(powNumbersArray)
```

Comment:
If you want just to print tuple here, there's no need to create an array here. Also there's no need to cast it to `[Any]`. If you want to store this new tuple in an array, you need to create a variable before and append a new value here